src/lib/markbind/src/lib/markdown-it/index.js

```diff
@@ -26,6 +26,19 @@ markdownIt.use(require('markdown-it-mark'))
   .use(require('markdown-it-task-lists'), {enabled: true})
   .use(require('markdown-it-linkify-images'), {imgClass: 'img-fluid'})
   .use(require('markdown-it-attrs'))
+  .use(require('markdown-it-regexp')(
+    /:(fa[brs]|glyphicon)-([a-z-]+):/m,
+    (match, utils) => {
+      let iconFontType = match[1];
+      let iconFontName = match[2];
+
+      if (iconFontType === 'glyphicon') {
+        return `<span aria-hidden="true" class="glyphicon glyphicon-${iconFontName}"></span>`;
+      } else {
+        return `<span aria-hidden="true" class="${iconFontType} fa-${iconFontName}"></span>`;
+      }
+    }
+  ))
```

Comment:
Let's abstract this into `./markdown-it-icons`.

Reply:
@acjh done!